### ISSUE TITLE
Clarify primary SPDX IDs for var/licenses/*.json

### ISSUE THREAD

**Issue by stefan6419846** [hesa/foss-licenses#194]
I am currently trying to match the licenses from `var/licenses/*.json` with the SCTK database. Doing so, I stumbled upon two files which cannot be mapped directly:

* LicenseRef-flame-x11-keith-packard, which is understandable as this uses your custom namespace.
* LicenseRef-scancode-docbook, which refers to SCTK. Nevertheless, there is an official SPDX ID for this now (DocBook-XML) according to https://scancode-licensedb.aboutcode.org/docbook.html, which is missing from https://github.com/hesa/foss-licenses/blob/main/var/licenses/LicenseRef-scancode-docbook.json as well. (While we are at it: DocBook might be a candidate for ambiguities as well due to the additional DocBook-Schema ID being available.)

This raises the question for me how one should interpret the primary SPDX IDs present in the JSON dataset provided here. Usually we want to map and verify this for a SBOM export (like when using `license-expression`, which itself is using the SCTK database), but this seems to only be possible with a well-known primary SPDX ID.

**Comment by hesa** [comment]
- LicenseRef-flame-x11-keith-packard - there is a not already on my laptop to clean up regarding x11-keith-packard. Will look in to this. I think this license should be merged with `HPND`

- LicenseRef-scancode-docbook - there was a recent update with a real SPDX id for LicenseRef-scancode-docbook which triggered a new file: https://github.com/hesa/foss-licenses/blob/main/var/licenses/DocBook-XML.json. Does this solve your problem.

**Comment by hesa** [comment]
_Note: assuming SCTK refers to scancode toolkit: foss-licenses' data need to be merged with SCTK._

**Comment by stefan6419846** [comment]
Yes, SCTK is ScanCode Toolkit, which basically acts as the data source for the ScanCode LicenseDB linked in my comment. Sorry for being a bit unspecific here.

Yes, the issue with `LicenseRef-scancode-docbook` has been resolved by matching both the database of SCTK and this repository by using the new `DocBook-XML` key as the file identifier here now.

Regarding `DocBook-Schema ID`: In this side note, I have been referring to the ambiguities at https://github.com/hesa/foss-licenses/blob/main/var/ambiguities.json The license `DocBook` might be a candidate for this, as there apparently are `DocBook-Schema` and `DocBook-XML` now and "DocBook" as a license being stated somewhere is not clearly mappable to one of them automatically.

**Comment by hesa** [comment]
This is what we have today

|SPDX id | Scancode DB id | 
|------------|------------------------|
| DocBook-XML | docbook |
| DocBook-Schema | docbook-schema |

Adding `DocBook` to ambiguities feels like the right thing to do (just as "GNU" alone is ambiguous), but it might create confusion having:
* `docbook` referring to `DocBook-XML` but 
* `DocBook` being ambiguous

**Comment by hesa** [comment]
Perhaps, we should compile a list of "DocBook" license names in this ticket and then look at the issue with that in mind?

**Comment by stefan6419846** [comment]
I am not completely sure about the best approach here or whether to take any action about this specific ambiguity here now - I just happened to stumble upon this as a side-effect of the actual issue. With me not having a strong opinion here, I am leaving this up to you being the maintainer and knowing the data the best.

**Comment by hesa** [comment]
spdx.org:
* https://spdx.org/licenses/DocBook-Schema.html
* https://spdx.org/licenses/DocBook-Stylesheet.html
* https://spdx.org/licenses/DocBook-XML.html

scancode-licensedb.aboutcode.org:
* https://scancode-licensedb.aboutcode.org/docbook.html (DocBook-XML)
* https://scancode-licensedb.aboutcode.org/docbook-schema.html (DocBook-Schema)
* https://scancode-licensedb.aboutcode.org/docbook-stylesheet.html (DocBook-Stylesheet)
